packages/format-library/src/text-color/index.js

```diff
@@ -87,7 +87,10 @@ function TextColorEdit( {
 					onClose={ disableIsAddingColor }
 					activeAttributes={ activeAttributes }
 					value={ value }
-					onChange={ onChange }
+					onChange={ () => {
+						onChange( ...arguments );
+						disableIsAddingColor();
+					} }
```

Comment:
The `arguments` object isn't available in arrow functions, so what you're passing here is actually the props. Use [rest parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters) instead.

```suggestion
					onChange={ ( ...args ) => {
						onChange( ...args );
						disableIsAddingColor();
					} }
```

Reply:
To be honest, I tested locally as: 
```javascript
onChange={ ( e ) => { 
    onChange( e ); 
    disableIsAddingColor();
} }
```
And later changed to `...arguments` 😅

Reply:
I didn't know it wasn't available in arrow functions. Or I forgot it... 😅 
Also, I tested it with `...arguments`, and it looked like it worked... But I just tested again, and it's not. 
Probably forgot to build or something, since I wasn't running `npm run dev`...

Anyway, thanks. I'll correct it right away.